packages/slice-machine/package.json

```diff
@@ -16,6 +16,8 @@
     "dev-cypress": "concurrently \"next dev\" \"npm run dev --prefix ../../e2e-projects/cypress-next-app\"",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
     "prepublishOnly": "npm run build",
+    "storybook": "STORYBOOK_DISABLE_TELEMETRY=1 storybook dev --no-open --port 6006",
```

Comment:
⚠️ To make the package.json cleaner, I would but the telemetry param in the .storybook/main.js file: https://storybook.js.org/docs/react/configure/telemetry#how-to-opt-out